docs/multi-node.qmd

```diff
@@ -2,11 +2,12 @@
 title: Multi Node
```

Comment:
```suggestion
title: Distributed Finetuning For Multi-Node with Axolotl and FSDP
```

Reply:
@winglian Committed The Changes you suggested.